opentelemetry-api/src/logs/mod.rs

```diff
@@ -0,0 +1,64 @@
+//! # OpenTelemetry Logs API
+
+use crate::ExportError;
+use futures_channel::{mpsc::TrySendError, oneshot::Canceled};
+use std::time::Duration;
+use thiserror::Error;
+
+/// Describe the result of operations in log SDK.
+pub type LogResult<T> = Result<T, LogError>;
+
+#[derive(Error, Debug)]
+#[non_exhaustive]
+/// Errors returned by the log SDK.
```

Comment:
I think this should be in the SDK only. Its probably a mistake that was already in traces api. See https://github.com/open-telemetry/opentelemetry-rust/issues/1042